pages.py

```diff
@@ -2,15 +2,22 @@
 from ._builtin import Page, WaitPage
 from .models import Constants
 
+
+class NotMobilePlayers(WaitPage):
```

Comment:
You can't have this here. Grouping is done purposefully in creating_session, because agents need to be paired with each other according to getting the probability of same group/other group.

Why do you need to separate mobile players, and how would this waitpage do that?

Reply:
No problem. I'll erase that